cas-server-documentation/installation/LDAP-Authentication.md

```diff
@@ -10,12 +10,12 @@ LDAP integration is enabled by including the following dependency in the Maven W
 <dependency>
      <groupId>org.jasig.cas</groupId>
      <artifactId>cas-server-support-ldap</artifactId>
-     <version>${cas.version}</version>
+     <version>${project.version}</version>
```

Comment:
@PetrGasparik It wont compile because that is expected of you to define. See https://github.com/Jasig/cas-overlay-template/blob/master/pom.xml#L79